app/src/main/java/de/westnordost/streetcomplete/quests/collection_times/NumberSystem.java

```diff
@@ -0,0 +1,109 @@
+package de.westnordost.streetcomplete.quests.collection_times;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+/** aka number range / Zahlenraum */
+public class NumberSystem
```

Comment:
You can reuse this too. If you just copy the files without making a change, it just adds too much complexity and it is really hard to maintain...